lib/gtfs/timepoint.ex

```diff
@@ -3,5 +3,32 @@ defmodule Gtfs.Timepoint do
   In GTFS, timepoints are known as "checkpoints"
   """
 
+  alias Gtfs.Csv
+  alias Gtfs.Trip
+
   @type id :: String.t()
+
+  @type id_set :: MapSet.t(id())
+
+  @spec includes_a_checkpoint_and_in_id_set?(Csv.row(), id_set) :: boolean
+  def includes_a_checkpoint_and_in_id_set?(timepoint_row, id_set),
+    do: includes_a_checkpoint?(timepoint_row) && in_id_set?(timepoint_row, id_set)
+
+  @spec trip_timepoints_from_csv([%{optional(String.t()) => String.t()}]) ::
```

Comment:
This can use the new `Csv.row()`

Reply:
The only difference here is `optional` vs `required`. Is this ever really `optional`? Just want to double check.

Reply:
I guess the difference is whether a csv with no columns is a csv. I don't think it matters.